### ISSUE TITLE
CJ4: Fuel Management Page 2 wants attention badly

### ISSUE THREAD

**Issue by J-Hoskin** [Working-Title-MSFS-Mods/fspackages#256]
When switching from page 2 on fuel management FMS screen, to any other, it switches back to page 2.

**Comment by dga711** [comment]
Yeah. It's the page refresh timers. When you hit a bad timing this happens with all auto refreshing pages, i think.

We need to find a better concept for these timers. Which goes hand in hand with page architecture.

**Comment by Wombii** [comment]
in CJ4_FMC_FuelMgmtPage.js I added "return true;" on line 232 and it seems to have stopped the problem of the fmc fighting itself for me. (using release 0.6.0)
from:
`fmc.registerPeriodicPageRefresh(() => {
            FuelMgmtPage2Instance.update();
 }, 1000, true);`

to:
`fmc.registerPeriodicPageRefresh(() => {
            FuelMgmtPage2Instance.update();
            return true;
}, 1000, true);`

**Comment by dga711** [comment]
hmm @Wombii 
The return true would indicate to the refresh to automatically set a new refresh timer again.
Since we already set the refresh again ourselves in the update() method, this should lead to multiple timers runnning.
Just wondering why this doesn't worsen the problem for you 🤔 

We had to resort to the method of setting the refresh ourselves because the automatic one introduced problems with the call stack on another page.
